### ISSUE TITLE
1.10.2 in Unstable 4.2.4 Curse pack  Mechanical user doesn't work when is placed in claimed/loaded chunk

### ISSUE THREAD

**Issue by diegofpais** [rwtema/extrautilities_old_issues#941]
Mechanical user doesn't work when is placed in claimed chunk (FTB utilities) or loaded chunk (chunk loading ward). Works normally when chunk loaders are not active.

**Comment by charlesbevans** [comment]
Confirmed in a modified version of the "All the Mods" modpack using FTB utilities. It runs with the chunk claimed, but not with the chunkloader on, even if I'm standing in the chunk.

**Comment by Try2bcool69** [comment]
This still has this problem in the Hermitpack 1.10.2 (as of v 1.4.0)  Works fine in Wilderness chunks, but not in claimed ones.

Found the solution to this, there is a setting to enable/disable fake players in loaded chunks, I would think it would be in the team config tab (which is not currently working for me), but you can change it directly with: 
/ftb team config ftbu.fake_players true

**Comment by ghost** [comment]
Yeah, but it still doesn't work when you're standing next to it. Maybe this should be sent over to the FTBU team.

**Comment by ktole1999** [comment]
does mechanical miner use fake player to mine blocks?

**Comment by ghost** [comment]
Apparently. I unloaded the chunk and suddenly the blocks broke. The /ftb command also did the trick.

**Comment by ktole1999** [comment]
have you tried another mod quarry/miner to see if it works?

**Comment by ktole1999** [comment]
so after you made the setting change you had to reload the chunk. _makes a mental note_

**Comment by ghost** [comment]
Nope. I was standing right there. Hit enter and bam, no more cobblestone in front of the miner.
